mease_lab_to_nwb/convert_ced/convert_ced.py

```diff
@@ -8,19 +11,44 @@
 ced_file_path = base_path / "m365_pt1_590-1190secs-001.smrx"
 nwbfile_path = base_path / "CED_stub.nwb"
 
+# Enter Session and Subject information here
+session_description = "Enter session description here."
+
+# Manually insert the session start time
+session_start = datetime(1971, 1, 1, 1, 1, 1)  # (Year, Month, Day, Hour, Minute, Second)
```

Comment:
```suggestion
session_start = datetime(1970, 1, 1)  # (Year, Month, Day)
```

Reply:
There appears to be a known windows specific issue where `datetime(1970,1,1)` can't be used with `.astimezone()`, so I incremented it to 1971. Should I just remove the `astimezone` instead?